addons/event/data/email_template_data.xml

```diff
@@ -281,5 +281,159 @@
             ]]></field>
         </record>
 
+        <record id="event_change_notification" model="mail.template">
+            <field name="name">Event: schedule change</field>
+            <field name="model_id" ref="event.model_event_registration"/>
+            <field name="email_from">${(object.event_id.organizer_id.email or object.event_id.user_id.email or '')|safe}</field>
+            <field name="email_to" >${object.partner_id.email or object.email|safe}</field>
+            <field name="lang">${object.partner_id.lang}</field>
+            <field name="reply_to">${object.event_id.reply_to|safe}</field>
+            <field name="subject">
+%if object.event_id.state == 'confirm'
+Schedule of event ${object.event_id.name} changed.
+%endif
+%if object.event_id.state == 'cancel'
+Event ${object.event_id.name} is cancelled.
+%endif
+</field>
+            <field name="body_html"><![CDATA[
```

Comment:
With the new sanitizer options this will lead to issues. Your mako should be contained inside valid xml.